vss-extension.json

```diff
@@ -1,159 +1,204 @@
+                "id": "username",
+                "name": "Username",
+                "description": "Username",
+                "inputMode": "textbox",
+                "isConfidential": false,
+                "validation": {
+                  "isRequired": false,
+                  "dataType": "string",
+                  "maxLength": 300
                 },
-                "authenticationSchemes": [
-                    {
-                        "type": "ms.vss-endpoint.endpoint-auth-scheme-basic",
-                        "inputDescriptors": [
-                            {
-                                "id": "username",
-                                "name": "Username",
-                                "description": "Username",
-                                "inputMode": "textbox",
-                                "isConfidential": false,
-                                "validation": {
-                                    "isRequired": false,
-                                    "dataType": "string",
-                                    "maxLength": 300
-                                },
-                                "values": {
-                                    "inputId": "username",
-                                    "isDisabled": true,
-                                    "defaultValue": ""
-                                }
-                            },
-                            {
-                                "id": "password",
-                                "name": "Personal access token",
-                                "description": "Team Services personal access token. Requires at least the Marketplace (publish) scope.",
-                                "inputMode": "passwordbox",
-                                "isConfidential": true,
-                                "validation": {
-                                    "isRequired": true,
-                                    "dataType": "string",
-                                    "maxLength": 300
-                                }
-                            }
-                        ]
-                    }
-                ]
-            }
-        }
-    ]
+                "values": {
+                  "inputId": "username",
+                  "isDisabled": true,
+                  "defaultValue": ""
+                }
+              },
+              {
+                "id": "password",
+                "name": "Personal access token",
+                "description": "Team Services personal access token. Requires at least the Marketplace (publish) scope.",
+                "inputMode": "passwordbox",
+                "isConfidential": true,
+                "validation": {
+                  "isRequired": true,
+                  "dataType": "string",
+                  "maxLength": 300
+                }
+              }
+            ]
+          }
+        ]
+      }
+    },
+    {
+      "id": "vsts-marketplace-endpoint-type",
```

Comment:
Probably there is a missing commit that you forgot to include in the PR. You cannot have duplicate contribution Ids. When I tried to publish this in the public gallery I got an error
```
The vsomanifest file contains one or more validation errors:The same 'id' property value 'bruca.vsts-developer-tools-build-tasks.vsts-marketplace-endpoint-type' was supplied for multiple contributions.
```

Could you check what is missing?